### DISCUSSION TITLE
Best Camera/Battle Switch Settings Recommendations?

### DISCUSSION THREAD

**Discussion by TopGear11TC** [MerlinCooper/iRacingReplayDirector#115] [General]
I've been using this program on and off for a few years, but it seems like recently it wants to jump around more between cars on track instead of sticking to battles which makes me wonder if it has more to do with the replay file from iRacing...

But anyway...

I'm doing both Sports Car and Oval race replays and I'm trying to find that sweet spot that sticks with battles that are close of course, but to also do a little look through the field on occasion.

My YouTube channel https://www.youtube.com/@11collier11 has several to view, but the latest ones just aren't working as well as the earlier ones, and again I don't know if it's more to do with the settings in IRR, or if it's something that's change in iRacing's replays (or even my system).

So I'd love to see a list of recommended settings from users, maybe even by track or even car/track?

**Comment by chcornell25649** [comment]
I know this is fairly old but have recently started using iRacing Replay Director. My community of Racers loves that I can get the videos out so fast but does not like the overlay so much. But as far as the Camera settings I kind of default to the basics.
Incident camera is Far Chase
percentages are as follows
tv1 15
tv2 15
tv3 15 
tv4 if present 5
gyro if on oval 10
cockpit or roll bar (depending on the view you like 10
Blimp 5
chopper 10
far chase 5 
rear chase 5

I am starting to test out different camera angles with rear views and gearboxes or suspensions. these can be good for dirt ovals.

hope this helps
if you want to see how they turn out https://www.youtube.com/@BackdraftSeven

**Reply by TopGear11TC** [reply]
You can set the overlays to None and use something like RaceLabs or other 3rd party tools.  iOverlay works ok, but you can't use Standings as it will stick to the final results of the race instead of changing in real time.

Here's the last one I made:  https://www.youtube.com/watch?v=DFi_ZziyvE4

**Reply by chcornell25649** [reply]
What is your process for not letting IRD encode the video? I tried the fast video record and it only makes a 2 min video. Only thing i can do is just  kill the program when it goes to encode.

**Reply by TopGear11TC** [reply]
I'm using OBS to capture the replay based on the hotkey assignments in both OBS and IRD. When OBS stops recording, IRD shows a failed encoding, but I don't care about that.  I only want to take the MKV file made by OBS and upload to YouTube.

I can recall years ago when I needed it to do the encoding to get the overlays embedded and create an MP4 file, but using OBS lets me have control of overlays, recording quality and so on.

Also, I never use the fast replay recording.  I will just let it sit and run itself for however long it takes to get the real time recording at a very high bitrate quality in OBS.

**Reply by chcornell25649** [reply]
I appreaciate the information. This is the outcome using SDK. 
https://youtu.be/LOS6p7pfrj8?si=pRyjIk5vmje92GTL

**Reply by TopGear11TC** [reply]
That looks great! The image quality is top notch, and that overlay is really nice!

I have a set of custom cameras for tracks and cars that I've collected over the years, nothing taken from pay sites or anything, just free postings, and most are really good. You can download them from this link:  https://drive.google.com/file/d/1XtsnIWbjzGs3ezxpWScClUfTbh09c8uW/view?usp=drive_link

Just copy into your cameras folder in Documents/iRacing and then use this quick video I made recently:

https://youtu.be/ON_sBt6wX24

Best of luck!

**Reply by chcornell25649** [reply]
Thank you for the video. That is very useful.
Also not sure if you are looking for any league racing but we have a really good group of people that we are looking to expand.
